dspace-server-webapp/src/main/java/org/dspace/app/rest/model/AuditEventRest.java

```diff
@@ -0,0 +1,192 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.app.rest.model;
+
+import java.util.Date;
+import java.util.UUID;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonProperty.Access;
+import org.dspace.app.rest.RestResourceController;
+
+/**
+ * The Audit Event REST Resource.
+ *
+ * @author Andrea Bollini (andrea.bollini at 4science.it)
+ */
+@LinksRest(links = {
+    @LinkRest(method = "getEperson", name = AuditEventRest.EPERSON),
+    @LinkRest(method = "getObject", name = AuditEventRest.OBJECT),
+    @LinkRest(method = "getSubject", name = AuditEventRest.SUBJECT)
+})
+public class AuditEventRest extends BaseObjectRest<UUID> {
+    public static final String NAME = "auditevent";
+    public static final String NAME_PLURAL = "auditevents";
+    public static final String CATEGORY = RestAddressableModel.SYSTEM;
+
+    public static final String EPERSON = "eperson";
+    public static final String SUBJECT = "subject";
+    public static final String OBJECT = "object";
+
+    private UUID epersonUUID;
+    private UUID objectUUID;
+    private String objectType;
+    private UUID subjectUUID;
+    private String subjectType;
+    private String eventType;
+    private Date timeStamp;
+    private String detail;
+    private String metadataField;
+    private String value;
+    private String authority;
+    private Integer confidence;
+    private Integer place;
+    private String action;
+    private String checksum;
+
+    @Override
+    @JsonProperty(access = Access.READ_ONLY)
+    public String getType() {
+        return NAME;
+    }
+
+    @Override
+    public String getTypePlural() {
+        return NAME_PLURAL;
+    }
+
+    @Override
+    public String getCategory() {
+        return CATEGORY;
+    }
+
+    @Override
+    public Class getController() {
+        return RestResourceController.class;
+    }
+
+    public UUID getEpersonUUID() {
```

Comment:
Don't want to be picky about it, but when writing code, it's helpful if the IDE suggests and displays some information regarding the available class methods, even if we consider them to be self explanatory, by adding java docs to it, even if they are just gets and sets, could be helpful to understand what it's expected.